shell/components/CruResource.vue

```diff
@@ -587,76 +587,74 @@ export default {
               </template>
             </template>
             <template #controlsContainer="{showPrevious, next, back, activeStep, canNext, activeStepIndex, visibleSteps}">
-              <template name="form-footer">
```

Comment:
That's something that has to be investigated because unless the custom footer has been removed along the way, it must have been added for sure somewhere.

Reply:
> That's something that has to be investigated because unless the custom footer has been removed along the way, it must have been added for sure somewhere.

The other instances of `form-footer` appear to be direct descendants of CruResource. In the description for this PR, it was mentioned that this behavior could be related to a change in how Vue3 deals with nested slots, or Vue3 is more strict about matching slots. 

I've created a barebones example to demonstrate this behavior[^1]:

```
<script setup>
import CruResource from './CruResource.vue';
import Wizard from './Wizard.vue';
</script>

<template>
  <CruResource>
    <template #cru-header>
      <h1>CRU HEADER</h1> 
    </template>
    <template #cru-body>
      <h2>CRU BODY</h2>
      <Wizard>
        <template #wiz-body>
          <h3>WIZ BODY</h3>
        </template>
        <template #wiz-footer>
          <template #cru-footer>
            <h3>NOT FAIL</h3>
          </template>
        </template>
      </Wizard>
    </template>
  </CruResource>
</template>
```

In the example above, `#cru-footer` is nested under `#wiz-footer`. This is invalid because the Wizard is unable to match a slot that named `cru-footer` and nothing renders (the template fails to compile in the example, but the overall concept appears to be the same). The component will render as expected we reorganize our slot usage so that `#cru-footer` is a child of `CruResource`.

I'm unable to locate a specific mention of this change in behavior in the vue 3 migration guide. If I had to guess, I think it would be a knock-on effect related to this change[^2]:

> `<template>` tags with no special directives (v-if/else-if/else, v-for, or v-slot) are now treated as plain elements and will result in a native `<template>` element instead of rendering its inner content.

[^2]: https://v3-migration.vuejs.org/breaking-changes/#other-minor-changes

[^1]: https://play.vuejs.org/#eNqtU11PwjAU/StNffBFtyj6opMEESPGiEGN0Swxc7tAdWuXfiBq+O+2Kx0bDHzxrfeec889be/9wZ0896YK8AkORMxJLpEAqfJ2SEmWMy5Rl6shCKZ4DGjEWYZ2Pb+SM8W7pyX7iXxHPCmJNnScwLcttLgOJGR5GknQEUJBRbJI6JQjoJ2Yq/0JRAnwBabRyUG7O3xEV73ORW8Y+DpEizq/prwm9MaSr4rMYSFzPrh41iKHS8BaL+OazCf5rstYqVb7qf/ipFrV0lVLDXojxmTlfg3GGxi26+3gAV12+jcrXTf1XU8Gi49yD1ZnBNX/1pkajPewFDGjIzL23gWjepB+TFGIY5blJAU+yCVhVIT4BBWIwaI0ZZ/XRU5yBXsuH08g/mjIv4uZyYX4joMAPoUQl5iM+BikhXv3tzDT5xLMWKJSzd4Cmpulyni0tHNFE227wivc9osBJ3T8IHozCVS4Sxmjhjkv+CHWw97dcvWl3ZZ3VNSFdK5fcWWn1hZy2/IkZOq+TqRMIhplcKZfs1ybECN/E8MM8jbcjl3JCHzbbXUKlqv+H9bNRjjrjc7cCm5yvlypP52/ToGb39S+W96xd4Tnv2TSnN8=

Reply:
The generated error in Vue SFC Playground is so confusing 😬

Reply:
Ok, I mixed up when the named slot is defined and when it's used.
`name="form-footer"` and `#form-footer` mean the same.